legacy_packages/cli/src/cli/index.ts

```diff
@@ -250,15 +250,12 @@ const main = async () => {
     )
     .option("--app", "Create a web3 app.")
     .option("--contract", "Create a web3 contract project")
-    .option("--ts, --typescript", "Initialize as a TypeScript project.")
-    .option("--js, --javascript", "Initialize as a JavaScript project.")
     .option("--forge", "Initialize as a Forge project.")
     .option("--hardhat", "Initialize as a Hardhat project.")
     .option("--extension", "Create a smart contract extension.")
     .option("--react", "Initialize as a Create React App project.")
     .option("--next", "Initialize as a Next.js project.")
     .option("--vite", "Initialize as a Vite project.")
-    .option("--reactNative", "Initialize as a React Native project.")
     .option("--express", "Initialize as a Express project.")
     .option("--node", "Initialize as a Node project.")
```

Comment:
the node one was heplful for scripting but we don't have it right now so would prob remove too